pitch.md

```diff
@@ -57,13 +125,38 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+This project is pretty large. I feel like there is a huge amount of work to be
+done in language design. Figuring out how to integrate lots of different methods
+of robot control and allow users to easily specify logic without limiting them
+sounds like a difficult challenge. Implementation will probably also require a
+bit of work, but I would hazard that about 80\% of this project is just figuring
+out what is good for the users. The language has to be as big as a GPL, but this
+is easy if the DSL is internal.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a pretty big project. I'm hoping to make something that will be
+personally useful to me and potentially useful to others doing robotics. As I
+mentioned before, there are lots of other languages that try to do similar
+things to what I'm doing, and it will probably take lots of work to figure out a
+solution that I find better than theirs. If this DSL ends up being external,
+implementation will probably also require a lot of work since it needs to be as
+powerful as a GPL, but if it is internal the implementation will not be too
+difficult. Since the language design is so much of the project, I feel that if I
+get a design I'm satisfied with, I'm OK with leaving it largely unimplemented or
+"faking it" for the final.
 
 ### Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
 
+This is a good project because it is useful and has a huge language design
+component. A warning sign may be if I start implementing too early. If I don't
+think hard enough about the design, I could end up implementing something that
+is no better than current solutions and have to redesign, wasting the
+implementation effort. If I find myself wanting to start implementing, I'll try
+writing sample programs and see what I find frustrating about that, and then go
```

Comment:
I really like this idea! I think I'll adopt it as a test to pass before I establish any sort of interface.